From bd938c052369c051dc7ffdf7c8bb72d4c3fa4d74 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Mon, 6 Jan 2025 17:51:42 +0100
Subject: [PATCH 718/841] feat(st): update ST specific version
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

NOT_UPSTREAMABLE

update version to r2.0

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git cb1c325fa6b9e73fe2569ffc3f75df6439d07815 feat(st): update ST specific version]
Change-Id: I6870db82e11eb2a7152ea77e6de8c9ae3181203c
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/427131
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 plat/st/common/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/common/common.mk b/plat/st/common/common.mk
index 186bf12e9..066d1faad 100644
--- a/plat/st/common/common.mk
+++ b/plat/st/common/common.mk
@@ -11,7 +11,7 @@ STM32MP_RECONFIGURE_CONSOLE	?=	0
 STM32MP_UART_BAUDRATE		?=	115200
 
 # Add specific ST version
-ST_VERSION 			:=	r1.0
+ST_VERSION 			:=	r2.0
 ST_GIT_SHA1			:=	$(shell git rev-parse --short=8 HEAD 2>/dev/null)
 VERSION_STRING			:=	v${VERSION_MAJOR}.${VERSION_MINOR}-${PLAT}-${ST_VERSION}(${BUILD_TYPE}):${BUILD_STRING}(${ST_GIT_SHA1})
 
-- 
2.47.2

